var/spack/repos/builtin/packages/psrcat/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Psrcat(MakefilePackage):
+    """A proper description of your package."""
+
+    homepage = "https://www.atnf.csiro.au/research/pulsar/psrcat/"
+    url = "https://www.atnf.csiro.au/research/pulsar/psrcat/downloads/psrcat_pkg.v1.68.tar.gz"
+
+    version("1.68", sha256="fbe4710c9122e4f93dbca54cf42cc2906f948f76885b241d1da2f8caecfbc657")
+
+    def build(self, spec, prefix):
+        makeit = which("./makeit")
+        makeit()
+
+    def install(self, spec, prefix):
+        bindir = prefix + "/bin/"
+        mkdir(bindir)
+        copy("psrcat", bindir)
```

Comment:
```suggestion
        mkdirp(prefix.bin)
        install("psrcat", bindir)
```
Prefix has convenience attributes for subdirectories, and the `install` method manages installation permissions in addition to copying the files.